p5js/FaceApi/FaceApi_Video_Expressions/sketch.js

```diff
@@ -0,0 +1,56 @@
+let faceapi;
+let video;
+let detections;
+
+// by default all options are set to true
+const detection_options = {
+    withLandMarks: false,
+    withFaceExpressions: true,
+    withFaceDescriptors: false,
+}
+
+function setup() {
+    createCanvas(400, 240);
+
+    // load up your video
+    video = createCapture(VIDEO);
+    video.size(width, height);
+    // video.hide(); // Hide the video element, and just show the canvas
+    faceapi = ml5.faceApi( video, detection_options, modelReady)
+    textAlign(RIGHT);
+}
+
+function modelReady() {
+    console.log('ready!')
+    console.log(faceapi)
+    
+    faceapi.detect( gotResults)
+
+}
+
+function gotResults(err, result) {
+    if (err) {
+        console.log(err)
+        return
+    }
+    // console.log(result)
+    detections = result;
+
+    background(255);
+    if (detections) {
+        if (detections.length > 0) {
```

Comment:
Is there a way we can simplify this for the user (this actually comes up with a lot of other ml5 functions, e.g. image classification where I am often writing `if (results && results.length > 0)`. Maybe we don't even trigger the callback if there are no results? Or we have some sort of property like `results.success`? This probably belongs in its own issue / discussion.

Reply:
@shiffman - oh this is definitely something for a larger discussion. Let's put a pin in this for now 📍